### DISCUSSION TITLE
Clearing the background color can cause the loss of materials in glb/gltf models. However, vsgt models remain unaffected.

### DISCUSSION THREAD

**Discussion by WdsPjm** [vsg-dev/VulkanSceneGraph#871] [General]
auto commandGraph = vsg::createCommandGraphForView(window, camera, vsg_scene);
        auto secondary_view = vsg::View::create(camera, vsg_scene);
        auto secondary_RenderGraph = vsg::RenderGraph::create(window, secondary_view);
        secondary_RenderGraph->clearValues[0].color = {{0.2f, 0.2f, 0.2f, 1.0f}};
        commandGraph->addChild(secondary_RenderGraph);

![1688728636763](https://github.com/vsg-dev/VulkanSceneGraph/assets/52599568/4de77428-32d8-42c9-9ede-a8d5c38e6821)

**Comment by robertosfield** [comment]
There isn't much we suggest with such snippet of code, all we can do is suggest further investigation at your end.

First thing to do is to enable the Vulkan debug layer to see if there are any errors reported.

Also try to see if you can get any of the vsgExamples programs to exhibit this issue, that gives others a chance of reproducing the problem,
